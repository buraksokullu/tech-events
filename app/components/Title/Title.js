import React from 'react';
import PropTypes from 'prop-types';
import s from './Title.scss';

const Title = ({ children, className }) => (
  <h1 className={`${s.titleHolder} ${className}`}>{children}</h1>
);

Title.propTypes = {
  children: PropTypes.node.isRequired,
  className: PropTypes.string
};

Title.defaultProps = {
  className: undefined
};
export default Title;
