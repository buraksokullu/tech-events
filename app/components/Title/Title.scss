.titleHolder {
  font-family: $font-family-secondary;
  font-size: 28px;
  font-weight: 600;
  line-height: 1.5;
  letter-spacing: 0.33px;
  margin: 0;
  &.marginTop24 {
    margin-top: 24px;
  }
}
