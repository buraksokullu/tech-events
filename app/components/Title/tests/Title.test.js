import React from 'react';
import { shallow } from 'enzyme';
import Title from '../Title';

describe('Title', () => {
  it('Should render correctly', () => {
    const component = shallow(<Title>Title</Title>);

    expect(component).toMatchSnapshot();
  });
});
