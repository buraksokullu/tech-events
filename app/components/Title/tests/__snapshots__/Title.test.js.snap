// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Should render correctly 1`] = `
<h1
  className="undefined undefined false"
>
  Title
</h1>
`;
